support/provenance/provenance.py

```diff
@@ -102,54 +103,61 @@ def run(
 
     host = HOST(cluster_nodes, password, base_url, username, verify_host_certs)
 
-    provenance = trawl_registry(host)
-    updates = get_historic(provenance, reset)
+    extant_lidvids = get_extant_lidvids(host)
+    updates = get_successors_by_lidvid(extant_lidvids)
 
     if updates:
-        update_docs(host, updates)
+        write_updated_docs(host, updates)
 
     log.info('completed CLI processing')
 
 
-def get_historic(provenance: {str: str}, reset: bool) -> {str: str}:  # TODO: populate comment and rename for clarity
-    log.info('starting search for history')
+def get_successors_by_lidvid(extant_lidvids: Iterable[str]) -> Mapping[str, str]:
+    """
+    Given a collection of LIDVIDs, return a new mapping to their updated direct successors.
+    """
+
+    log.info('Generating updated history...')
 
-    log.info('   reduce lidvids to unique lids')
-    lids = sorted({lidvid.split('::')[0] for lidvid in provenance})
+    unique_lids = {lidvid.split('::')[0] for lidvid in extant_lidvids}
 
-    log.info('   aggregate lidvids into lid buckets')
-    aggregates = {lid: [] for lid in lids}
-    for lidvid in provenance: aggregates[lidvid.split('::')[0]].append(lidvid)
+    log.info('   ...binning LIDVIDs by LID...')
```

Comment:
Me being pedantic: use an actual ellipsis character `…` instead of three periods `...` 😅